pkgs/development/node-packages/composition.nix

```diff
@@ -6,7 +6,7 @@
 
 let
   nodeEnv = import ./node-env.nix {
-    inherit (pkgs) stdenv python2 util-linux runCommand writeTextFile;
+    inherit (pkgs) stdenv python2 utillinux runCommand writeTextFile;
```

Comment:
Not sure about this change...

Reply:
This change breaks eval. If you update to the latest master you should get node2nix 1.9.0 which fixes this.